src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        this.color = colorSupplier.getRandomColor();
+    }
+
+    public void setNumber(int num) {
+        Random random = new Random();
+        this.number = random.nextInt(num);
+    }
```

Comment:
```suggestion
    public void setNumber(int num) {
        this.number = num;
    }
```
you should get a number here and just set it